docs/transpile/representing_quantum_computers.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Representing quantum computers
+description: The necessary backend information for successfully transpiling quantum circuits
+
+---
+
+# Representing quantum computers
+
+To construct an equivalent circuit that can run on a specific backend, the transpiler needs certain details about the backend.  Typically, this information is found in the `backend` or `target = backend.target` class, so you don't need to pass anything further to the transpiler. (TODO: check with Matthew about how this is moving, seems there is no `target` attribute for older `Backend` classes). However, if more infomation is provided, the transpiler can use it to try to produce the best circuit to run on that hardware.  
+
+<Admonition>
+Because many of the underlying transpilation algorithms are stochastic, there is no guarantee that a better circuit will be found. 
+</Admonition>
+
+<span id="default-config"></span>
+## Default configuration
+
+The simplest use of the transpiler is to provide all the backend infomation by providing the `backend` or `target` (TODO: only available on newer backends?). To better understand how the transpiler works, construct a circuit and transpile it with different information:
+
+Import the necessary libraries and instantiate the backend:
+
+```python
+    from qiskit import QuantumCircuit, transpile
+    from qiskit.providers.fake_provider import FakeSherbrooke
+
+    backend = FakeSherbrooke()
+    target = backend.target
+```
+
+Constuct a 12-qubit circuit with circular entanglement:
+
+```python
+    qc = QuantumCircuit(12)
+    qc.h(0)
+    _ = [qc.cx(idx, idx+1) for idx in range(11)]
+    qc.cx(11, 0)
+    qc.draw('mpl')
+```
+![The 12-qubit test circuit](/images/transpile/qc-circular.png)
+
+Transpile the circuit with the simplest options. This example uses `optimization_level=2` to highlight a feature:
+
+```python
+    qc_t_target = transpile(qc, target=target, optimization_level=2, seed_transpiler=12345)
+    qc_t_target.draw('mpl', style='iqp', idle_wires=False)
+```
+![The transpiled 12-qubit test circuit using Target information](/images/transpile/qc_t_target.png)
```

Comment:
For images, it's best to please follow the file name structure of `/images/transpile/file-name/my_img.png`, e.g. `/images/transpile/representing-quantum-computers/qc_t_target.png`

That keeps the images better organized and more clear what is what.